mdbenchmark/analyze.py

```diff
@@ -177,19 +98,26 @@ def analyze(directory, plot, ncores, output_name):
     df.to_csv(output_name)
 
     if plot:
+        console.warn(
+            'This feature is now outdated.'
+            'Please use mdbenchmark plot to access all plotting features')
+
+        fig = Figure()
+        FigureCanvas(fig)
+        ax = fig.add_subplot(111)
+
         df = pd.read_csv(output_name)
 
-        # We only support plotting of benchmark systems from equal hosts /
-        # with equal settings
-        uniqueness = df.apply(lambda x: x.nunique())
-        if uniqueness['gromacs'] > 1 or uniqueness['host'] > 1:
-            console.error(
-                'Cannot plot benchmarks for more than one GROMACS module '
-                'and/or host.')
+        df_sel = 'nodes'
```

Comment:
What do we need this for? Any reason to set the variable here, instead of in the `plot_line` call further below?